### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by liyingxin97**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,21 +27,34 @@ To run the tests for the service, from the command line run:
...
     * `POST /customers`
-    * The JSON request body for creating a customer is expected to contain the following keys: ``first_name, last_name, user_name, addresses, password``
+    * The JSON request body for creating a customer is expected to contain the following keys: ``first_name, last_name, user_name, password，addresses, street_address, state, city, and zipcode``
     * The expected types of the values for each of the expected keys are as follows:
         - first_name: string
         - last_name: string
         - user_name: string
-        - addresses: array of string
         - password: string 
+        - address: array of string
```
Comment (ryanjcohen) [inline README.md:36]: there is no longer an array of strings

Comment (Bobby-Zien) [reply README.md]: Yes

**Comment by ryanjcohen** [comment]
I think the readme is still missing 2 endpoints that I believe exist:  GET /customers/{customer_id}/addresses and GET /customers/{customer_id}/addresses/{address_id} - not sure if we want to open a new ticket to fix this for the next sprint because we're running out of time or if we want to get those changes in quickly

**Comment by Bobby-Zien** [comment]
> I think the readme is still missing 2 endpoints that I believe exist: GET /customers/{customer_id}/addresses and GET /customers/{customer_id}/addresses/{address_id} - not sure if we want to open a new ticket to fix this for the next sprint because we're running out of time or if we want to get those changes in quickly

Yes, let's, merge for now.

**Review by Bobby-Zien — APPROVED** [review]

**Pull request merged**